### PR TITLE
Adding custom wiringPi PKGBUILD

### PR DESCRIPTION
**Pull request by lonewsk**

### CODE REVIEW

**Comment by css459** [comment]
Will test this today and get it merged. Looks great and thanks again!!

**Comment by lonewsk** [comment]
Hope it'll work. I got audio covered also :)

**Comment by css459** [comment]
Looks great thank you and sorry for the delay :)

File: audio-clockworkpi-a06/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+pkgname=audio-clockworkpi-a06
+pkgver=1.0
+pkgrel=1
+pkgdesc="ClockworkPi DevTerm A06 Gearbox"
+arch=('aarch64')
+license=('LGPLv2')
+depends=('alsa-lib')
+groups=('clockworkpi-a06')
+source=("git://github.com/maxfierke/clockworkpi-a06-post-install")
```
Comment (maxfierke) [inline audio-clockworkpi-a06/PKGBUILD:11]: Please use the Manjaro ARM repos as the upstream for this. My repo is not necessarily up-to-date with them.

**Comment by css459** [comment]
@lonewsk 

Do you get the following error when attempting to build the wiringpi-library?

```
[Compile] piFace.c
piNes.c:25:10: fatal error: wiringPi.h: No such file or directory
   25 | #include <wiringPi.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:75: piNes.o] Error 1
make: *** Waiting for unfinished jobs....
gertboard.c:41:10: fatal error: wiringPi.h: No such file or directory
   41 | #include <wiringPi.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:75: gertboard.o] Error 1
piFace.c:31:10: fatal error: wiringPi.h: No such file or directory
   31 | #include <wiringPi.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:75: piFace.o] Error 1
maxdetect.c:30:10: fatal error: wiringPi.h: No such file or directory
   30 | #include <wiringPi.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:75: maxdetect.o] Error 1
ds1302.c:30:10: fatal error: wiringPi.h: No such file or directory
   30 | #include <wiringPi.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:75: ds1302.o] Error 1
==> ERROR: A failure occurred in package().
    Aborting...
```

File: audio-clockworkpi-a06/PKGBUILD (hunk 1 above)
Comment (css459) [reply audio-clockworkpi-a06/PKGBUILD:11]: Thanks will fix this shortly

**Comment by lonewsk** [comment]
No, I can't reproduce this error. Did you used `makepkg -si`?

Can you show me what's in your src/ directory?

I may have tried to install the wiring-git repo prior to making this PKGBUILD, I may have forgotten something.

**Comment by css459** [comment]
I tried this within a clean Arch ARM image when trying to build the next release.  The src directory contains only, in full, the `DevTerm` repository.

**Pull request merged**